packages/material-ui/src/styles/transitions.d.ts

```diff
@@ -38,8 +26,3 @@ export interface TransitionsOptions {
   ) => string;
   getAutoHeightDuration?: (height: number) => number;
 }
-
-// export type TransitionsOptions = DeepPartial<Transitions>;
-
-declare const transitions: Transitions;
-export default transitions;
```

Comment:
Default export is gone but was never part of the public interface (only first and second level imports are public).